sympy/polys/rootisolation.py

```diff
@@ -1835,14 +1982,24 @@ def conjugate(self):
 
     def is_disjoint(self, other):
         """Return ``True`` if two isolation intervals are disjoint. """
-        if self.conj != other.conj:
+        if isinstance(other, RealInterval):
+            return other.is_disjoint(self)
+        if self.conj != other.conj:  # above and below real axis
             return True
         re_distinct = (self.bx <= other.ax or other.bx <= self.ax)
         if re_distinct:
             return True
         im_distinct = (self.by <= other.ay or other.by <= self.ay)
         return im_distinct
 
+    def _set_imaginary(self):
+        a1, a2 = self.a
+        b1, b2 = self.b
+        a = a1 - a1, a2
```

Comment:
Will this be `0, a2`?

Reply:
It will be a `PythonRational(0)`, I believe. I was just avoiding creating it explicitly since PythonRational already knows how to do subtraction.

Reply:
I thought so. It could probably also be GMPYRational. How about adding a comment?